esp/src/src-react/routes.tsx

```diff
@@ -72,6 +72,7 @@ const routes: Routes = [
         children: [
             { path: "", action: (context) => import("./components/Files").then(_ => <_.Files filter={parseSearch(context.search) as any} />) },
             { path: "/legacy", action: () => import("./layouts/DojoAdapter").then(_ => <_.DojoAdapter widgetClassID="DFUQueryWidget" />) },
+            { path: "/:Name", action: (ctx, params) => import("./components/FileDetails").then(_ => <_.FileDetails logicalFile={params.Name as string} />) },
```

Comment:
This won't work with the "tabs" paradigm as you end up with `/:Name/:Tab` which is the same as `/:NodeGroup/:Name`